.vscode/settings.json

```
@@ -0,0 +1,6 @@
+{
+  "editor.tabSize": 4,
+  "prettier.useTabs": false,
+  "prettier.tabWidth": 4,
+  "editor.formatOnSave": true
```

```suggestion
  "editor.formatOnSave": true,
  "editor.defaultFormatter": "esbenp.prettier-vscode"
```

I suggest prettier as a default formatter.